README.tfdoc.hcl

```diff
@@ -171,18 +206,55 @@ section {
           default     = true
         }
 
-        variable "module_tags" {
-          type        = map(string)
+        variable "module_timeouts" {
+          type        = any
+          readme_type = "object(google_monitoring_notification_channel)"
           description = <<-END
-            A map of tags that will be applied to all created resources that accept tags. Tags defined with 'module_tags' can be overwritten by resource-specific tags.
+            How long certain operations (per resource type) are allowed to take before being considered to have failed.
           END
-          default     = {}
+          default = {}
           readme_example = <<-END
-            module_tags = {
-              environment = "staging"
-              team        = "platform"
+            module_timeouts = {
+              google_monitoring_notification_channel = {
+                create = "4m"
+                update = "4m"
+                delete = "4m"
+              }
             }
           END
+
+          attribute "google_monitoring_notification_channel" {
+            type        = any
+            readme_type = "object(timeouts)"
+            description = <<-END
+              Timeout for the `google_monitoring_notification_channel` resource.
+            END
+            default =  null
```

Comment:
we do not document null   @thiesen can we automatically exclude rendering it if it is `null`

```suggestion
```

Reply:
resolved in 217df72c